cocos/core/scene-graph/index.ts

```diff
@@ -27,8 +27,8 @@ import './node-event-processor';
 import './deprecated';
 import './deprecated-3.7.0';
 // @deprecated since v3.7, please use Node instead.
-export { Node as BaseNode } from './node'; //reserve BaseNode for compatibility
 export { Node } from './node';
+export { Node as BaseNode } from './node'; //reserve BaseNode for compatibility
```

Comment:
It is better to add comment that, should export it after export Node.

Reply:
good idea.